packages/flutter/lib/src/widgets/scroll_physics.dart

```diff
@@ -362,7 +362,8 @@ class BouncingScrollPhysics extends ScrollPhysics {
 ///    behavior.
 ///  * [GlowingOverscrollIndicator], which is used by [ScrollConfiguration] to
 ///    provide the glowing effect that is usually found with this clamping effect
-///    on Android.
+///    on Android. By default, [GlowingOverscrollIndicator]'s glow adopts the
+///    [ThemeData.accentColor] in a Material Design app.
```

Comment:
That's not true. GlowingOverscrollIndicator has no default color. By default, MaterialApp configures a GlowingOverscrollIndicator to use the accent color.

Reply:
Specifically referred to the [MaterialApp] in comment.